src/runtime_src/core/common/api/xrt_kernel.cpp

```diff
@@ -159,6 +159,29 @@ debug_cmd_packet(const std::string& msg, const ert_packet* pkt)
   return fnm;
 }
 
+std::string
+cmd_state_to_string(ert_cmd_state state)
+{
+  static const std::map<ert_cmd_state, const char*> ert_cmd_state_string {
+   {ERT_CMD_STATE_NEW, "ERT_CMD_STATE_NEW"},
```

Comment:
I guess that all these top-level uppercase symbols comes from the compatibility with C headers too?

Reply:
Yes.  :-(